src/esn.calendar.libs/app/components/event/form/event-form.controller.js

```diff
@@ -80,11 +80,18 @@ function CalEventFormController(
   $scope.submitSuggestion = submitSuggestion;
   $scope.onDateChange = onDateChange;
   $scope.hideEventForm = true;
+  $scope.dragModal = dragModal;
 
   // Initialize the scope of the form. It creates a scope.editedEvent which allows us to
   // rollback to scope.event in case of a Cancel.
   $scope.initFormData();
 
+  function dragModal() {
+    $('.modal-dialog').draggable({
```

Comment:
You ask jQuery to lookup the whole DOM for any element that has the '.modal-dialog' class. Can you be more specific ? Can't you for example launch the .draggable call on dialog creation, in the component root directive ?